functions/Repair-DbaOrphanUser.ps1

```diff
@@ -206,12 +207,20 @@ function Repair-DbaOrphanUser {
 
                             #With the collection complete invoke remove.
                             if ($RemoveNotExisting -eq $true) {
-                                if ($Pscmdlet.ShouldProcess($db.Name, "Remove-DbaOrphanUser")) {
+                                if ($Force -eq $true) {
```

Comment:
No need to do `$force -eq $true` here, you can just do `($force)` 😃

Reply:
Thanks @shaneis , is this the same too

if ($RemoveNotExisting -eq $true) ?

Reply:
I think it works in more scenarios than just true / false but I just use it in those scenarios. Yeah, it works there as well 👍